etrago/tools/utilities.py

```diff
@@ -444,26 +444,25 @@ def set_q_national_loads(self, cos_phi=1):
     national_buses = network.buses[
         (network.buses.country == "DE") & (network.buses.carrier == "AC")
     ]
-
-    network.loads_t["q_set"].loc[
-        :,
-        network.loads.index[
-            network.loads.bus.astype(str).isin(national_buses.index)
-        ].astype(int),
-    ] = network.loads_t["p_set"].loc[
+    
+    # Calculate q national loads based on p and cos_phi
+    new_q_loads = network.loads_t["p_set"].loc[
         :,
         network.loads.index[
             network.loads.bus.astype(str).isin(national_buses.index)
```

Comment:
q is now calculated for all loads attached to 'AC' buses. That can have some side-effects: e.g. e-Mobility is a load attached to the AC nodes in the lowflex scenario but not in eGon2035. If we keep it like this, e-Mobility has a q demand in some scenarios, but not in all.

Reply:
Fixed in b0fb292aa49241c015f6e2ba9b9384e36cfdd21b